pre_commit/error_handler.py

```diff
@@ -4,12 +4,14 @@
 
 import contextlib
 import os.path
+import sys
 import traceback
 
 import six
 
 from pre_commit import five
 from pre_commit import output
+from pre_commit.constants import VERSION as pre_commit_version
```

Comment:
iirc the rest of the codebase uses `import pre_commit.constants as C` and then `C.VERSION`